addons/web_editor/models/image_data.py

```diff
@@ -0,0 +1,71 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ImageData(models.Model):
+    _name = 'web_editor.image.data'
+    _description = 'Image Data'
+
+    res_model = fields.Char('Resource Model', required=True, readonly=True)
+    res_field = fields.Char('Resource Field', required=True, readonly=True)
+    res_id = fields.Many2oneReference('Resource ID', model_field='res_model', required=True, readonly=True)
+
+    original_id = fields.Many2one('ir.attachment', string='Original (unoptimized, unresized) attachment')
+    original_src = fields.Char(compute='_compute_original_src')
```

Comment:
Yes, definitely a `related` one. Concerning the fact that is needed, I would like to have your opinion: I could do without having it in this model and add `image_data_dict[‘original_src’] = self[‘original_id’].image_src` in `_get_image_data()` but I feel it would maybe be clearer to have the `original_src` related field stored in the `web_editor.image.data` model as it is an important part of the image data information? Not sure what is the best to do here.